module/VuFind/src/VuFind/Session/Database.php

```diff
@@ -29,6 +29,9 @@
 
 namespace VuFind\Session;
 
+use VuFind\Db\Service\ServiceAwareInterface;
```

Comment:
Since we're eventually going to migrate all the session handlers to Doctrine, maybe it would make more sense to add the ServiceAwareInterface to AbstractBase instead of to Database, and then you'll have access to the extra methods in the right place when you need them in future.